log4j-jdbc/pom.xml

```diff
@@ -30,89 +30,99 @@
   <name>Apache Log4j JDBC</name>
   <description>Apache Log4j Java Database Connectivity (JDBC).</description>
 
-  <properties>
-    <log4jParentDir>${basedir}/..</log4jParentDir>
-
-    <!--
-      ~ OSGi and JPMS options
-      -->
-    <bnd-extra-package-options>
-      <!-- `log4j-jndi` is optional -->
-      org.apache.logging.log4j.jndi;resolution:=optional
-    </bnd-extra-package-options>
-  </properties>
-
   <dependencies>
+
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-core</artifactId>
+      <artifactId>log4j-api</artifactId>
     </dependency>
+
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-jndi</artifactId>
-      <optional>true</optional>
+      <artifactId>log4j-core</artifactId>
     </dependency>
+
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-api-test</artifactId>
-      <scope>test</scope>
+      <artifactId>log4j-plugins</artifactId>
     </dependency>
+
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-core-test</artifactId>
       <scope>test</scope>
+      <!--
+        ~ Exclude some unused dependencies of `log4j-core-test`.
```

Comment:
Why do we need this?

Reply:
Right, we don't. The `log4j-core-test` artifact has a lot of dependencies that are added to the test classpath of each of our components. We probably should remove those that are not necessary.

Fixed in https://github.com/apache/logging-log4j2/pull/2915/commits/c9cc897bd2142eded6dfa475446cbe805064e1a8

Reply:
Removed from the wrong module.

Reply:
Fixed in af3b21b067a2546f089716d16a2a57ee0292d517.